docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md

```diff
@@ -0,0 +1,736 @@
+With this new model, you now have a clear distinction between the people that are allowed to access the administration panel, and the users of the application you are building with Strapi.
+
+More practically, it means that you will have a new strapi_administrator collection that will be created automatically in your database. 
+On startup, this table will be empty so, when migrating from alpha to beta, you can either create this new administrator user with the registeration page OR manually migrate your previous users with `administrator` role.
+
+### Cleaning up the `users-permissions.users` collection
+
+If you only used the `administrator` role to give access to the admin panel to certain users, and never used this role for your application business logic; you can delete the role.
+
+If you haven't created any relation with the `User` model in your `Content Types` and don't use those users in your application business logic; you can remove every user you have migrated to the `strapi_administrator` collection.
+
+
+Finally, if you have chosen to migrate your previous admin users in the new `strapi_administrator` collection and your `User` model has at least one relation with another model you may need to keep both collections in sync 
+
+
+**Example**: Some of your application users can edit their profile and access the admin panel. If theyy change their email you need to make sure their `administrator` entity also changes email.
+::: warning
+We really recommend separating you users from your administrators to avoid this situation which is not a good practice.
+:::
+
+
+## Running your migrated project
+
+To run your migrated project you will now need to run `strapi develop` or `npm run develop` to run the project in watch mode (e.g auto reloading on content-type creation).
+
+To run strapi without watch mode then run `strapi start` or `npm run start`
+
+Finally if you want to run your project in different envrionments use `NODE_ENV=env npm run start`
+
+## Updating Deployments
+
+### Building your admin panel
+
+If you are deploying your Strapi project with the Admin panel together, you will have to make sure the `build` script runs before running the project. (e.g `npm run build`)
+
+Depending on your deployment environment you might have to run the `build` script with `NODE_ENV=production npm run build` if the envrionment doesn't include it by default.
+
+For example `heroku` will run the `build` script for you and set `NODE_ENV=production` before running the scripts so you won't have to think about it.
+
+### Running the project
+
+Previously you could run your project by running `node server.js`. As we remove the `server.js` file you will have to either run `npm run start` or create a `server.js` file (read more [here](#migrating-server.js))
```

Comment:
Previously, you could run your project by running `node server.js`. The beta version removes the `server.js` file, and so you will have to either run `npm run start` or manually create a `server.js` file (read more [here](#migrating-server.js))